romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
Because it is no sense to load full lodash library for only one usage.
This regex very simple, i don't think that it is difficult to understand.
If u want, u can use something like:
`bemjson.trim().splice(0, -1)`
and that should be valid because more than one `;` should be wrong case.